src/Locale/uk/cake.po

```diff
@@ -0,0 +1,272 @@
+# LANGUAGE translation of CakePHP Application
```

Comment:
Shouldn't this file be on `ua` folder? [`UA`](https://en.wikipedia.org/wiki/ISO_3166-2:UA) being the 2 letter-code for Ukraine.